.github/actions/spelling/allow/names.txt

```diff
@@ -69,8 +70,8 @@ Wirt
 Wojciech
 zadjii
 Zamor
-zamora
 Zamora
+zamora
```

Comment:
The sort command I'm using isn't stable for case. So you may see jitter in things like this.